api/app/Http/Resources/UserStubResource.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+// Sometimes we have to query for relationship non-nullable fields when they're not needed because of codegen
```

Comment:
hopefully we can finally not need this :cry:

Reply:
Yeah, this is a real thorn...